src/screens/ContactScreen/index.js

```diff
@@ -307,7 +307,13 @@ const ContactScreen = ({navigation}) => {
           }}
           rowRenderer={rowRenderer}
           scrollViewProps={{
-            refreshControl: <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
+            refreshControl: (
+              <RefreshControl
+                tintColor={COLORS.white2}
+                refreshing={refreshing}
+                onRefresh={onRefresh}
+              />
+            )
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `tintColor` prop has been added to the `RefreshControl` component and set to `COLORS.white2`. This change will make the refresh indicator white. Please ensure that this color choice doesn't affect visibility against the background color of the scroll view.
```diff
- refreshControl: <RefreshControl refreshing={refreshing} onRefresh={onRefresh} />
+ refreshControl: (
+   <RefreshControl
+     tintColor={COLORS.white2}
+     refreshing={refreshing}
+     onRefresh={onRefresh}
+   />
+ )
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->